### ISSUE TITLE
Missing `alt` on Google Maps image

### ISSUE THREAD

**Issue by aardrian** [bpetetot/conference-hall-beta#858]
On the page: https://conference-hall.io/public/event/FdP7ePMBrbLq9kZd4Hye

* The Google logo inside the maps embed has no `alt` attribute:
```
<img src="https://maps.gstatic.com/mapfiles/embed/images/google4.png" style="width: 52px;">
```

For more information: WCAG 2.1 Success Criterion [1.1.1 Non-text Content, Level A](https://www.w3.org/WAI/WCAG21/quickref/#non-text-content)

This is absolutely a picayune issue, but if the Google Maps embed is there solely to show the entirety of a country for a virtual event, consider removing it and replacing it with a static image. Less for users to download, less for the browser to parse, less third-party tracking.

**Comment by bpetetot** [comment]
We not only display the country with Google Map. It depends on the event address given by the organizers. (it can be a country, a city or a more precise location)

But you're totally right about static image of maps. We don't use it because the pricing is much higher than embeded map (almost free). We will study alternative to Google Maps in order to get static map generation. I close the issue for now

Thanks for the feedback.
